python/Ice/async/Client.py

```diff
@@ -5,7 +5,7 @@
 #
 # **********************************************************************
 
-import sys, os, traceback, threading, Ice
+import signal, sys, os, traceback, threading, Ice
 
```

Comment:
signal is not used in this file, why do we need the import?

Reply:
Thanks, I will fix it.